### PR TITLE
Decoding for staking options instructions

### PR DESCRIPTION
**Pull request by brittcyr**

### CODE REVIEW

File: components/instructions/programs/dual.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+              decodedInstructionData.optionExpiration.toNumber() * 1000
+            ).toDateString()}
+          </div>
+          <div>
+            Subscription Period End:{' '}
+            {new Date(
+              decodedInstructionData.subscriptionPeriodEnd.toNumber() * 1000
+            ).toDateString()}
+          </div>
+          <div>Num Tokens: {tokenAmount.toNumber()}</div>
```
Comment (DonDuala) [inline components/instructions/programs/dual.tsx:81]: Will this be in tokens or atoms or options contracts? Rename Num Option Tokens?

File: components/instructions/programs/dual.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    ],
+    getDataUI: async (connection: Connection, data: Uint8Array) => {
+      const soHelper = new StakingOptions(connection.rpcEndpoint)
+      const decodedInstructionData = new BorshInstructionCoder(
+        soHelper.getIdl()
+      ).decode(Buffer.from(data))?.data as initStrikeInstruction
+
+      return (
+        <div className="space-y-3">
+          <div>Strike: {decodedInstructionData.strike.toNumber()}</div>
```
Comment (DonDuala) [inline components/instructions/programs/dual.tsx:106]: Similar with strike, maybe clarify units?

File: components/instructions/programs/dual.tsx (hunk 1 above)
Comment (brittcyr) [reply components/instructions/programs/dual.tsx:81]: The calculation above makes this tokens

File: components/instructions/programs/dual.tsx (hunk 2 above)
Comment (brittcyr) [reply components/instructions/programs/dual.tsx]: added units. currently it matches the idl exactly

**Review by DonDuala — APPROVED** [review]
Just clarify units all around

**Pull request merged**